05-specifying-sample-designs.Rmd

```diff
@@ -1 +1,705 @@
+
+```{r}
+#| label: brr
+#| eval: FALSE
+des_brr <- dat %>%
+  as_survey_rep(weights = WT, repweights= all_of(str_c("REPWT", 1:20)),
+                type="BRR", mse=TRUE)
+
+des_brr <- dat %>%
+  as_survey_rep(weights = WT, repweights= starts_with("REPWT"),
+                type="BRR", mse=TRUE)
+```
+
+If the replicate weight variables are on the file consecutively, the following syntax can also be used:
+
+```{r}
+#| label: brr2
+#| eval: FALSE
+des_brr <- dat %>%
+  as_survey_rep(weights = WT, repweights= REPWT1:REPWT20, type="BRR",
+                mse=TRUE)
+```
+
+Typically, the replicate weights sum to a value similar to the main weight as they are both supposed to provide population estimates. Rarely, an alternative method will be used where the replicate weights have values of 0 or 2 in the case of BRR weights. In this case, the replicate weights are not combined and the option `combined_weights = FALSE` should be indicated with the default value being TRUE. This specific syntax is shown below:
+
+```{r}
+#| label: brr3
+#| eval: FALSE
+des_brr <- dat %>%
+  as_survey_rep(weights = WT, repweights= starts_with("REPWT"), 
+                type="BRR", combined_weights = FALSE, mse=TRUE)
+```
+
+#### Example {-} 
+
+The {survey} package includes a data example from Section 12.2 of Levy and Lemeshow. In this fictional data, two out of five ambulance stations were sampled from each of three emergency service areas (ESAs) thus BRR weights are appropriate with 2 PSUs (stations) sampled in each stratum (ESA). 
+
+```{r}
+#| label: brr_display
+data(scd)
+scdbrr <- scd %>%
+  as_tibble() %>%
+  mutate(
+    wt=5/2,
+    rep1 = 2 * c(1, 0, 1, 0, 1, 0),
+    rep2 = 2 * c(1, 0, 0, 1, 0, 1),
+    rep3 = 2 * c(0, 1, 1, 0, 0, 1),
+    rep4 = 2 * c(0, 1, 0, 1, 1, 0))
+
+scdbrr
+```
+
+In the code above, BRR weights are created as was done in Levy and Lemeshow. To specify the BRR weights, the following syntax is used:
+
+```{r}
+#| label: brr_des
+des_brr_scd <- scdbrr %>%
+  as_survey_rep(type = "BRR", repweights = starts_with("rep"),
+                combined_weights = FALSE, weight=wt)
+
+des_brr_scd
+
+summary(des_brr_scd)
+```
+
+Note that `combined_weights` was specified as `TRUE` because these weights are simply specified as 0 and 2 and do not incorporate the overall weight. When printing the object, the type of replication is noted as Balanced Repeated Replicates, the replicate weights are specified, and the weight variable. When looking at the summary, the only additional information provided are the variables included.
+
+### Fay's BRR Method
+
+Fay's BRR method for replicate weights still uses a Hadamard matrix to construct replicate weights but rather than deleting PSUs for each replicate, half of the PSUs have a replicate weight which is the main weight multiplied by $\rho$ and the other half have the main weight multiplied by $(2-\rho)$ where $0 \le \rho < 1$. Note that when $\rho=0$, this is equivalent to the standard BRR weights. 
```

Comment:
Do we want to add a note as to what rho is?

Reply:
@rpowell22  Let's discuss how we might clarify this and add more context.